doc/page-cmakefunctions.h

```diff
@@ -0,0 +1,123 @@
+
+facelift_add_qml_plugin(<TargetName>
+                        URI uri
+                        [VERSION version]
+                        [OUTPUT_BASE_DIRECTORY dir]
+                        [SOURCES src1 [src2...]]
+                        [HEADERS hdr1 [hdr2...]]
+                        [SOURCES_GLOB [Globbing expressions]...]
+                        [HEADERS_GLOB [Globbing expressions]...]
+                        [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                        [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                        [HEADERS_NO_INSTALL hdrn1 [hdrn2...]]
+                        [HEADERS_GLOB_NO_INSTALL [Globbing expressions]...]
+                        [HEADERS_GLOB_RECURSE_NO_INSTALL [Globbing expressions]...]
+                        [LINK_LIBRARIES lib1 [lib2...]]
+                        [NO_INSTALL])
+\endcode
+
+This function adds a Qml plugin.
+
+\code
+
+facelift_add_library(<TargetName>
+                     [STATIC | SHARED]
+                     [SOURCES src1 [src2...]]
+                     [HEADERS hdr1 [hdr2...]]
+                     [SOURCES_GLOB [Globbing expressions]...]
+                     [HEADERS_GLOB [Globbing expressions]...]
+                     [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                     [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                     [HEADERS_NO_INSTALL hdrn1 [hdrn2...]]
+                     [HEADERS_GLOB_NO_INSTALL [Globbing expressions]...]
+                     [HEADERS_GLOB_RECURSE_NO_INSTALL [Globbing expressions]...]
+                     [PUBLIC_HEADER_BASE_PATH path]
+                     [LINK_LIBRARIES lib1 [lib2...]]
+                     [NO_EXPORT]
+                     [NO_INSTALL])
+\endcode
+
+This function adds a library with the name \e TargetName from the specified source and header files.
+
+\code
+facelift_add_executable(<TargetName>
+                        [SOURCES src1 [src2...]]
+                        [HEADERS hdr1 [hdr2...]]
+                        [SOURCES_GLOB gsrc1 [Globbing expressions]...]
+                        [HEADERS_GLOB ghdr1 [Globbing expressions]...]
+                        [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                        [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                        [LINK_LIBRARIES lib1 [lib2...]])
+
+\endcode
+
+This function adds an executable with the name \e TargetName from the specified source and header files.
+
+\code
+facelift_add_test(<TargetName>
+                  [SOURCES src1 [src2...]]
+                  [HEADERS hdr1 [hdr2...]]
+                  [SOURCES_GLOB [Globbing expressions]...]
+                  [HEADERS_GLOB [Globbing expressions]...]
+                  [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                  [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                  [LINK_LIBRARIES lib1 [lib2...]])
+\endcode
+
+This function adds a test with the name \e TargetName.
+
```

Comment:
The following lines do not belong to the "facelift_add_test" section, which is confusing. Add a separator ?

Reply:
done